agent/backend/pktvisor/pktvisor.go

```diff
@@ -223,7 +223,7 @@ func (p *pktvisorBackend) Start(ctx context.Context, cancelFunc context.CancelFu
 	p.scraper.StartAsync()
 
 	if p.scrapeOtel {
-		p.scrapeOpenTelemetry(ctx)
+
```

Comment:
This should not be an empty block. It also looks like this PR stops using this config variable.

Reply:
fixed